benchmarks/benchmarks/optimize.py

```diff
@@ -209,7 +209,8 @@ def bench_run(self, x0, methods=None, **minimizer_kwargs):
         if methods is None:
             methods = ["COBYLA", 'Powell',
                        'L-BFGS-B', 'BFGS', 'CG', 'TNC', 'SLSQP',
-                       "Newton-CG", 'dogleg', 'trust-ncg', 'trust-region-exact']
+                       "Newton-CG", 'dogleg', 'trust-ncg', 'trust-trlib',
```

Comment:
Ok! Let us leave as it is for now... We may come back to this latter, if we have some better name in mind....